pkgs/desktops/gnome/extensions/systemd-manager/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "gnome-shell-extension-systemd-manager";
+  version = "16";
+
+  src = fetchFromGitHub {
+    owner = "hardpixel";
+    repo = "systemd-manager";
+    rev = "v${version}";
+    hash = "sha256-JecSIRj582jJWdrCQYBWFRkIhosxRhD3BxSAy8/0nVw=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/gnome-shell/extensions/
+    mv systemd-manager@hardpixel.eu $out/share/gnome-shell/extensions/
+  '';
```

Comment:
There are also [some Polkit files](https://github.com/hardpixel/systemd-manager#authorization) but we probably do not want to install them to default location since they loosen security. Maybe copy them to `share/doc/systemd-manager` directory for easier enabling and mention them in `longDescription`. Though the policy also FHS path so that would need to be patched.

Reply:
> I add the rule to `security.polkit.extraConfig` directly. Would it be better to source that file from this package directly?

I read this in the polkit service:

```nix
    # The polkit daemon reads action/rule files
    environment.pathsToLink = [ "/share/polkit-1" ];
```

I think it means that if the package would provide files in that directory - it should work if it the polkit service is enabled and the package is `environment.systemPackage`.

Reply:
Pushed a suggestion regarding the polkit policy.